lib/vagrant-service-manager/plugin.rb

```diff
@@ -1,3 +1,5 @@
+require_relative 'providers/docker'
```

Comment:
@bexelbie , @navidshaikh : 
Would it better if we use consistent naming? I would like to agree with the term `service`.
This line will be `require_relative 'services/docker'` or 
`Dir.glob('lib/*.rb') { |f| require_relative f }`(best I could) for automatically adding new services. 

Also, `Vagrantfile` line as:
`config.servicemanager.services = 'openshift'`([docker default hence this line will not be needed](https://goo.gl/0F6ttj)).